packages/camera/camera/example/pubspec.yaml

```diff
@@ -14,11 +14,28 @@ dependencies:
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
     path: ../
+  camera_platform_interface:
+    path: ../../camera_platform_interface
   flutter:
     sdk: flutter
   path_provider: ^2.0.0
   video_player: ^2.1.4
 
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
```

Comment:
remove

Reply:
Removed.
... This comment is managed by `dart run script/tool/bin/flutter_plugin_tools.dart`.